tests/group_tree.py

```diff
@@ -8,16 +8,16 @@ def setUp(self):
         self.norne = sunbeam.parse(norne, ('PARSE_RANDOM_SLASH', sunbeam.action.ignore))
 
     def test_group_tree(self):
-        self.gr_obj = self.norne.schedule.group_tree(0)
-        parent = self.gr_obj.parent('PROD')
-        children = self.gr_obj.children('PROD')
+        self.gr_obj = self.norne.schedule._group_tree(0)
+        parent = self.gr_obj._parent('PROD')
+        children = self.gr_obj._children('PROD')
```

Comment:
You should normally not test the `_xxx` functions as they are "private".

Could you add the test from the PR description as we discussed?